models/utils/cohort_id_format.py

```diff
@@ -0,0 +1,81 @@
+from typing import Iterable
+
+from api.settings import COHORT_CHECKSUM_OFFSET, COHORT_PREFIX
+from models.utils.luhn import luhn_compute, luhn_is_valid
+
+
+def cohort_id_format(cohort_id: int | str) -> str:
```

Comment:
MH: Looks like this file has not much unit test coverage, would be nice to see more tests.